### PR TITLE
Add example of multi-stack trust relationship

### PR DESCRIPTION
**Pull request by adamconnelly**
# Description of the change

I've added an example of how to use a `for_each` block along with the `spacelift_aws_integration_attachment_external_id` data source to generate the trust relationship for attaching a single integration to multiple stacks.

## Checklist

Please make sure that the proposed change checks all the boxes below before requesting a review:

- [x] I have reviewed the [guidelines for contributing](https://github.com/spacelift-io/user-documentation/blob/main/CONTRIBUTING.md) to this repository.
- [x] The preview looks fine.
- [x] The tests pass.
- [x] The commit history is clean and meaningful.
- [x] The pull request is opened against the `main` branch.
- [x] The pull request is no longer marked as a draft.
- [x] You agree to license your contribution under the [MIT license](https://github.com/spacelift-io/user-documentation/blob/main/LICENSE) to Spacelift.

If the proposed change is ready to be merged, please request a review from `@spacelift-io/solutions-engineering`. Someone will review and merge the pull request.

Thank you for your contribution! 🙇

### CODE REVIEW

File: docs/integrations/cloud-providers/aws.md [hunk 1]
```diff
@@ -242,6 +242,88 @@ resource "spacelift_aws_integration_attachment" "this" {
 !!! info
     Please always refer to the [provider documentation](https://github.com/spacelift-io/terraform-provider-spacelift) for the most up-to-date documentation.
 
+### Attaching a Role to Multiple Stacks
+
+The previous example explained how to use the `spacelift_aws_integration_attachment_external_id` data-source to generate the assume role policy for using the integration with a single stack, but what if you want to attach the integration to multiple stacks? The simplest option would be to create multiple instances of the data-source - one for each stack - but you can also use a Terraform `for_each` condition to reduce the amount of code required:
+
+```hcl
```
Comment (jmfontaine) [inline docs/integrations/cloud-providers/aws.md:249]:
```suggestion
```terraform
```

**Review by jmfontaine — CHANGES_REQUESTED** [review]
Just a minor formatting issue, otherwise it looks good to me. 👍

**Comment by adamconnelly** [comment]
@jmfontaine that's the problem fixed.

**Review by jmfontaine — APPROVED** [review]
LGTM 👍

**Pull request merged**